env/env.go

```diff
@@ -88,20 +79,30 @@ func PrepareHelp(config interface{}) (string, error) {
 	var helpAux []byte
 	configValue := reflect.ValueOf(config).Elem()
 	for i := 0; i < configValue.NumField(); i++ {
-		fieldName := configValue.Type().Field(i).Name
-		var snakeCase []byte
-		for i, c := range fieldName {
-			if i > 0 && c >= 'A' && c <= 'Z' {
-				snakeCase = append(snakeCase, '_')
-			}
-			snakeCase = append(snakeCase, byte(c))
+		confKey := getConfKey(configValue.Type().Field(i))
+		if confKey == "-" {
+			continue
 		}
+
 		prefix := ""
 		if goconfig.PrefixEnv != "" {
 			prefix = goconfig.PrefixEnv + "_"
 		}
-		helpAux = append(helpAux, []byte(prefix+strings.ToUpper(string(snakeCase)))...)
-		helpAux = append(helpAux, []byte("=value\n")...)
+		helpAux = append(helpAux, []byte(prefix+strings.ToUpper(confKey)+"=value\n")...)
```

Comment:
I never squeeze so much in one line if string concat in `go` is involved. If you want an alternative, this is how I would do it - more verbose (but your solution si fine):
```suggestion
                tmp := fmt.Printf("%s%s=value\n", prefix, strings.ToUpper(confKey))
		helpAux = append(helpAux, []byte(tmp)...)
```

Reply:
I see. Btw I assume you probably meant to use `Sprintf` rather than `Printf`

Reply:
Yeah, sorry, I possess strong C habits.